addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -272,9 +272,12 @@
         },
 
         send: async function (e) {
+            const $button = this.$el.find('.s_website_form_send, .o_website_form_send');
```

Comment:
I would need more info to give a correct review on this one.
How `send` can be called if there is no send button ? 
Do you have steps to reproduce on the runbot ?
What is `a657c778e74aa937126032b300945013615662f9`, I dont find odoo commit with this ID ?
Why do you target this specific version ?

Please try to give as much info as possible (you can add them in the commit message too), it really helps a lot to review efficiently :+1:

Reply:
Is it okay , or you need more information ?

Reply:
Seems wrong to leave the `a-submit` class on the button and prevent it to submit at the same time don't you think ?

Reply:
Actually the original fix was to change the template to use the snippet correctly but that would require to update all the instances of the users manually , so maybe we can change the template in master but for stable it's better that we add a diff in js

Reply:
The risk with your change is that any custo of the form with a `<button class="a-submit">` was submitting the form and with your change it will not anymore. (because it doesn't match `.s_website_form_send, .o_website_form_send`).

I will let @rdeodoo have a look as he touched something similar recently

Reply:
As far as I can search, before https://github.com/odoo/odoo/commit/14fc7f3c923847f5e052888c2cfda0bf92316972 the only way to get in the send method was to have an element `.s_website_form_send, .o_website_form_send`

But the `l10n_mx_edi_website_sale` is incompatible with that change (because it uses the snippet, but expects a browser form sending), so this seems safe to me but I'm not too knowledgeable.

Reply:
@Guillaume-gdi , I made an override for the widget in the specific modules , so probabely we don't need this change anymore.